doc/userguide/configuration/multi-tenant.rst

```diff
@@ -55,6 +56,36 @@ Settings:
     - vlan-id: 1112
       tenant-id: 3
 
+
+This example uses the VLAN pair selector:
+
+::
+
+  multi-detect:
+    enabled: yes
+    selector: vlan-pair
+    loaders: 3
+
+    tenants:
+    - id: 1
+      yaml: tenant-1.yaml
+    - id: 2
+      yaml: tenant-2.yaml
+    - id: 3
+      yaml: tenant-3.yaml
+
+    mappings:
+    - vlan-id: 1000
```

Comment:
How long til we'll need qinqinqinqinqinqinq?  I like the `qinq` name here where the value is an array.  Limited to 2 for now perhaps, but could grow.

Reply:
I like both ideas and I suggest `vlan-id-tuple: [1000, 1]` along with changing the selector name from `vlan-pair` to `vlan-tuple`

Reply:
It would be nice if the selector value, and the name of the field for that select could be the same, however we don't have that for `vlan` and `vlan-id` now.  If `vlan-tuple` it could be nice to have:
```
selector: vlan-tuple
mappings:
- vlan-tuple: [..]
```

Reply:
I like that suggestion @jasonish  -- will do.